shelley-ma/shelley-ma-test/cddl-files/shelley-ma.cddl

```diff
@@ -230,16 +230,16 @@ native_script =
   // script_all
   // script_any
   // script_n_of_k
-  // timelock_start
-  // timelock_expiry
+  // require_time_hereafter
+  // require_time_before
```

Comment:
Maybe it's just me, but I find these _significantly_ more tricky to understand!

Reply:
It is not just you, I found the former names a lot clear to understand what they were.

Reply:
Other people told me that `timelock_start` and `timelock_expiry` were confusing, I think because `timelock_start` does not refer to the point in time when funds first become "locked". the time inside the timelock interval is actually when the funds are "unlocked".